bot/exts/evergreen/emoji_count.py

```diff
@@ -80,10 +85,11 @@ async def ec(self, ctx: commands.Context, *, emoji: str = None) -> Optional[str]
         emoji_dict = self.emoji_list(ctx, emoji_dict)
 
         if len(emoji_dict) == 0:
-            embed = self.generate_invalid_embed(ctx)
+            embed, msg = self.generate_invalid_embed(ctx)
         else:
-            embed = self.embed_builder(emoji_dict)
-        await ctx.send(embed=embed)
+            embed, msg = self.embed_builder(emoji_dict)
+        msg = msg.split("\n")
+        await LinePaginator.paginate([line for line in msg], ctx=ctx, embed=embed)
```

Comment:
Why not make the embed builders return a Linepaginator, instead of embed + message. Just do the split and ect before returning.

Reply:
linepagintor only takes maxlines and maxsize